tools/runners/parallels.sh

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env bash
+
+#############################################################
+
+_tmp_dir="$TMP_PATH/prepared_env"
+_sdl_prepared=$_tmp_dir/sdl
+
+_path_sdl="$SDL_CORE"
+_path_atf="$ATF_PATH"
+_test_result_path="$REPORT_PATH_TS"
+_testfile="$TEST_TARGET"
+_path_sdl_api="$SDL_API"
+_path_3rd_party="$THIRD_PARTY"
+_save_sdl_log="$SAVE_SDL_LOG"
+_save_sdl_core_dump="$SAVE_SDL_CORE_DUMP"
+_copy_ts="$COPY_TS"
+_path_to_atf_parallels="$_path_atf/atf_parallels"
+_test_id_file=.test_id
+
+#############################################################
+
+_queue_reference=queue_ref.tmp.txt
+_queue=queue.tmp.txt
+
+_overall_test_number=0
+_total_passed=0
+_total_failed=0
+_total_aborted=0
+_total_skipped=0
+
+function prepare_num_of_workers {
+  local num=$(wc -l $_queue | awk '{print $1}')
+  echo "Number of scripts to execute: "$num
+  echo "Max number of jobs: "$JOBS
+  _number_of_workers=$JOBS
+  if [ "$num" -lt "$JOBS" ]; then
+    _number_of_workers=$num
+  fi
+
+  echo "Number of workers: "$_number_of_workers
+}
+
+function prepare_sdl {
+  #
+  #   Preparation for SDL.
+  #     1) SDL has to be copied to tmp dir
+  #     2) Copy libs to bin directory
+  #
+
+  if [ -d "$_sdl_prepared" ]; then
+    rm -r "$_sdl_prepared"
+  fi
+  mkdir $_sdl_prepared
+
+  cp -r $_path_sdl $_sdl_prepared
+
+  if [ ! -z "$_path_3rd_party" ]; then
+    LD_LIBRARY_PATH=$_path_3rd_party/lib:$_path_3rd_party/x86_64/lib:$LD_LIBRARY_PATH \
+    ldd $_sdl_prepared/bin/smartDeviceLinkCore \
+    | grep "$_path_3rd_party/lib\|$_path_3rd_party/x86_64/lib" \
+    | awk '{print $3}' \
+    | xargs -L1 -I LIB cp LIB $_sdl_prepared/bin
```

Comment:
@jacobkeeler I was able to reproduce this issue in case if `THIRD_PARTY_INSTALL_PREFIX`, `THIRD_PARTY_INSTALL_PREFIX_ARCH` and `LD_LIBRARY_PATH` are not set and SDL dependencies installed in system folder (`/usr/local/lib`).
Please try the fix in 96120e0